### PR TITLE
fix: Add missing SentrySdkInfo.h to umbrella header

### PR DESCRIPTION
**Pull request by bruno-garcia**
## :scroll: Description

Missing public file (it's in signatures of methods exposed but itself is not)

## :bulb: Motivation and Context
missing in the umbrella header

## :pencil: Checklist

- [ ] I reviewed the submitted code
- [ ] I added tests to verify the changes
- [ ] I updated the CHANGELOG
- [ ] I updated the docs if needed
- [ ] No breaking changes

### CODE REVIEW

File: Sources/Sentry/Public/Sentry.h [hunk 1]
```diff
@@ -31,3 +31,4 @@ FOUNDATION_EXPORT const unsigned char SentryVersionString[];
 #import "SentryStacktrace.h"
 #import "SentryThread.h"
 #import "SentryUser.h"
+#import "SentrySdkInfo.h"
```
Comment (bruno-garcia) [inline Sources/Sentry/Public/Sentry.h:34]:
```suggestion
#import "SentrySdkInfo.h"

```

Comment (bruno-garcia) [inline Sources/Sentry/Public/Sentry.h:34]:
```suggestion
#import "SentrySdkInfo.h"

```

Comment (bruno-garcia) [reply Sources/Sentry/Public/Sentry.h:34]: doesn't want to add an empty line

**Comment by bruno-garcia** [comment]
Good bot

**Review by philipphofmann — APPROVED** [review]
Thanks a lot, LGTM

**Pull request merged**